File: examples/node/get-vehicle-history.js
import { apibaraGet } from "./apibara-client.js";

const vinOrLot = process.argv[2] || "WBAVL1C58FVY28848";
const data = await apibaraGet(`/vehicles/${encodeURIComponent(vinOrLot)}/history`, { per_page: 20 });

console.log(JSON.stringify(data, null, 2));

File: examples/python/get_vehicle_history.py
import sys
from pprint import pprint
from apibara_client import apibara_get

vin_or_lot = sys.argv[1] if len(sys.argv) > 1 else "WBAVL1C58FVY28848"
history = apibara_get(f"/vehicles/{vin_or_lot}/history", {"per_page": 20})

pprint(history)
